Dockerfile

```diff
@@ -1,48 +1,45 @@
 # ==============================
-FROM helsinkitest/python:3.9-slim as appbase
+FROM registry.access.redhat.com/ubi8/python-39 as appbase
```

Comment:
Is there a reason for not using ubi9?

Reply:
no reason, this is just copied from an another deckerfile.
I updated to ubi9